stubs/setuptools/pkg_resources/__init__.pyi

```diff
@@ -4,12 +4,14 @@ import zipimport
 from _typeshed import Incomplete
 from abc import ABCMeta
 from collections.abc import Callable, Generator, Iterable, Sequence
-from typing import IO, Any, TypeVar, overload
-from typing_extensions import Self, TypeAlias
+from re import Pattern
```

Comment:
Do these changes really belong in this PR?

Reply:
Nope, it's meant to be split off in #9747